rest/src/main/java/org/opentestsystem/authoring/testspecbank/client/tib/ValidationErrorCode.java

```diff
@@ -0,0 +1,9 @@
+package org.opentestsystem.authoring.testspecbank.client.tib;
+
+/**
+ * Possible validation error codes returned the code of a {@link tds.common.ValidationError}.
+ */
+public class ValidationErrorCode {
+    public static final String TEST_SPECIFICATION_NOT_FOUND = "notFound";
+    public static final String TEST_SPECIFICATION_EXCEPTION = "internalServerError";
+}
```

Comment:
Since these values are internal to TSB (they're used to determine which HTTP status code to return in event of an error), I didn't see a need to add these to a TSB client library. The only consumer of the TSB client library is ART, and ART can use the HTTP status codes to determine the appropriate action.

I say all this because putting this class in the service package deviates from our normal practice of putting classes like this in a client library/package.